packages/@contentful--app-scripts/package.json

```diff
@@ -28,7 +28,7 @@
     "prettier": "prettier **/*.js --write --ignore-path .gitignore",
     "lint": "eslint ./lib",
     "lint:fix": "npm run lint -- --fix",
-    "test": "mocha ./lib/**/*.test.js",
+    "test": "mocha ./lib/**/*.test.js ./lib/*.test.js",
```

Comment:
Is this needed? The first glob should capture everything 🤔

Reply:
Somehow it wasn't picking it up when being in this directory, did it work for you?